ReactiveCocoa/Swift/Event.swift

```diff
@@ -175,6 +175,12 @@ public func sendError<Value, Err: ErrorType>(sink: Event<Value, Err>.Sink, _ err
 	sink(.Error(error))
 }
 
+/// Puts an `Error` event into the given sink.
+public func sendError<Value>(sink: Event<Value, NoError>.Sink, _ error: NoError) {
+	assertionFailure("Don't send an Error to a Signal of type NoError NoError!")
```

Comment:
This is less strict than before: it's now a *runtime* assertion instead of something we can *know at compile time*. That's why this method does not make much sense.